process-analyser/src/com/axonivy/solutions/process/analyser/bo/CustomFieldFilter.java

```diff
@@ -1,16 +1,25 @@
 package com.axonivy.solutions.process.analyser.bo;
 
+import java.time.LocalDate;
+import java.util.List;
 import java.util.Objects;
 
 import ch.ivyteam.ivy.workflow.custom.field.ICustomFieldMeta;
 
 public class CustomFieldFilter {
   private ICustomFieldMeta customFieldMeta;
   private boolean isCustomFieldFromCase;
+  private List<Object> customFieldValues;
+  private List<Object> availableCustomFieldValues;
+  private List<LocalDate> timeStampCustomFieldValues;
```

Comment:
timeStampCustomFieldValues => I think timestamp is one word, then we should not uppercase 'S'